ios/Classes/PMPlugin.m

```diff
@@ -44,15 +44,9 @@ - (void)onMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result {
   PMManager *manager = self.manager;
```

Comment:
Can we implement permissions according to the `permission_handler` plugin? See https://github.com/Baseflow/flutter-permission-handler/blob/master/permission_handler/ios/Classes/strategies/PhotoPermissionStrategy.m